docs: update metabase store UI paths (#75136)
metabase/metabase: docs/data-studio/transforms/transforms-overview.md

Before you can start writing transforms, you'll need to enable transforms in your Metabase instance.

> If you are on a Metabase Cloud plan, only people logged in with an email of a [Metabase Store admins](../../cloud/accounts-and-billing.md#add-managers) (not just Metabase _instance_ admins) can enable basic transforms, because transforms incur a cost per run on Metabase Cloud.

Reviewer: ```suggestion
If you're on a Metabase Cloud plan, only people logged in with an email of a [Metabase Store admins](../../cloud/accounts-and-billing.md#add-managers) (not just Metabase _instance_ admins) can enable basic transforms, because transforms incur a cost per run on Metabase Cloud.
```
